src/utils.ts

```diff
@@ -104,6 +104,12 @@ export function getTSConfig(globals, collectCoverage: boolean = false) {
 
     config.inlineSourceMap = true;
     config.inlineSources = true;
+
+    if (config.outDir) {
```

Comment:
I added the `if` because I followed the code for `config.sourceMap` which is pretty much the same thing. I can remove the `if` from both if you like?

I can move it outside the containing `if`, however only coverage seems to be affected, so there is not really a need to tamper with the users' config when `collectCoverage` is false?